tools/simulated_hybrid_file_system.cc

```diff
@@ -6,13 +6,12 @@
 #include "util/stop_watch.h"
 #ifndef ROCKSDB_LITE
 
-#include "tools/simulated_hybrid_file_system.h"
-
 #include <algorithm>
 #include <sstream>
 #include <string>
 
 #include "rocksdb/rate_limiter.h"
+#include "tools/simulated_hybrid_file_system.h"
```

Comment:
This one looks odd - usually the header file corresponding to the source file gets included first

Reply:
Yes. claim to clang-format?